commands/command2.cpp

```diff
@@ -162,7 +162,7 @@ int cmdTraffic(const std::shared_ptr<Player>& player, cmd* cmnd) {
     )
         return(0);
     
-    if(player->isEffected("hold-person") &&
+    if(player->getAsCreature()->isMagicallyHeld() &&
```

Comment:
What's the need to call `getAsCreature()` here? I see other usages of `player->isMagicallyHeld()`

Reply:
Doubt it's needed. Probably just did it out of habit at the time or something. I'll fix it.

Reply:
Got rid of getAsCreature() for this here and elsewhere in the code. Wasn't needed, like you thought.

Reply:
Redundant call removed.